Disable support requires.txt since packages express their dependencies
via Depends/Recommends already.
--- a/setup.py
+++ b/setup.py
@@ -9,7 +9,6 @@ setup(
     author_email="hello@pts-project.org",
     description="PTS privacy preserving telemetry",
     url="https://github.com/PiRogueToolSuite/pts-telemetry",
-    install_requires=requirements,
     packages=find_packages(),
     zip_safe=True,
     entry_points={
